vulpes/memory/gamestate/table.cpp

```diff
@@ -23,3 +23,21 @@ MemRef Table::new_at(MemRef id) {
 bool Table::remove(MemRef id) {
     return (datum_delete(this, id.raw) != 0xFFFFFFFF);
 }
+
+size_t Table::count() {
+    // Universal table element counting function.
+    // Made to work with any table.
```

Comment:
This might be a gripe for future you, but making function-describing comments block comments outside the function can allow documentation generators to read and print out docs for your code. I've never seen one for C/C++ that reads inner comments like this.
```
/* Universal table element counting function. Should work with any table. */
size_t Table::count() {
   ...
```